modules/sdk-unified-wallet/src/ecdsa-unified-wallet/ecdsaEVMUnifiedWallets.ts

```diff
@@ -72,4 +77,28 @@ export class EcdsaEVMUnifiedWallets extends UnifiedWallets {
     });
     return keychainsTriplet;
   }
+
+  async getCoinWalletById(id: string, coinName: string): Promise<Wallet> {
```

Comment:
Can we modify this method signature to something like:
```
await evmWallets.getUnifiedWalletById(id).coin("eth");
```

Reply:
Trying to follow how we access wallets else where in the SDK example:

```
  const walletInstance = await basecoin.wallets().get({ id: walletId });
```

Open to suggestions / pushback on this

Reply:
I see your point, but I don't believe this need to be similar as EVMUnifiedWallet is a group of coins having the same address. We already know what those coins are, so better to pass in the coin we want and get the specific wallet instead of making a 2-step process (getWallet, then getCoin). I believe our current 1 step is better (getEvmWalletBasedOnId but for that specific coin.